tests/v2/validation/pipeline/scripts/build_qase_auto_testrun.sh

```diff
@@ -3,4 +3,11 @@ set -e
 cd $(dirname $0)/../../../../../
 
   echo "building qase auto testrun bin"
-  env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o tests/v2/validation/testrun ./tests/v2/validation/pipeline/qase/testrun
\ No newline at end of file
+  env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o tests/v2/validation/testrun ./tests/v2/validation/pipeline/qase/testrun
+
+  if [ $? -ne 0 ]; then
+    echo "Failed to build Qase auto testrun binary" >&2
+    exit 1
+else
+    echo "Successfully built Qase auto testrun binary"
```

Comment:
Is this log necessary?

Reply:
yeah. at times we do not know if the binary was run or not and if this is logged would provide details about the binary. I would like to keep it.